appcues/src/main/java/com/appcues/ui/AppcuesOverlayViewManager.kt

```diff
@@ -120,6 +124,24 @@ internal class AppcuesOverlayViewManager(
         }
     }
 
+    private fun Activity.updateOverlayVisibility(isVisible: Boolean) {
+        getParentView().let { parentView ->
+            parentView.post {
+                findViewById<ComposeView>(R.id.appcues_overlay_view)?.let {
+                    it.visibility = if (isVisible) ViewGroup.VISIBLE else ViewGroup.GONE
```

Comment:
there is a method View.isVisible [true/false]

Reply:
ah yeah I wasn't sure if that set it GONE when false, but looking at the docs, it does - so that should work fine to simplify.

Reply:
updated in https://github.com/appcues/appcues-android-sdk/pull/447/commits/885c29baa8fa9d996c65cbc578a33643d4a7ac95